appservice/multiplexer.py

```diff
@@ -170,6 +160,22 @@ def new_session(self):
             self.wfile.write("creating container failed: ".encode())
             self.wfile.write(content)
 
+    def new_session(self):
+        sessionid = str(uuid.uuid4())
+
+        if os.path.exists(PODMAN_SOCKET):
+            self.new_session_podman(sessionid)
```

Comment:
I think it might make sense to handle writing the response back to be "generic" but that can be done when we actually have a k8s API sessions

Reply:
So `new_session_podman` would return `response.status >= 200 and response.status < 300` for examle.

Reply:
Good point -- the previous code was also a bit buggy. I reworked that, PTAL?